magpie/api/generic.py

```diff
@@ -44,8 +44,8 @@ def not_found_or_method_not_allowed(request):
 
     Without this fix, both situations return [404] regardless.
     """
-    # noinspection PyProtectedMember
-    if isinstance(request.exception, PredicateMismatch) and request.method not in request.exception._safe_methods:
+    if isinstance(request.exception, PredicateMismatch) and \
+            request.method not in request.exception._safe_methods:  # noqa: W0212
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Access to a protected member _safe_methods of a client class](https://app.codacy.com/app/fmigneault/Magpie/pullRequest?prid=4650806)